src/quo/components/utilities/social/view.cljs

```diff
@@ -0,0 +1,61 @@
+(ns quo.components.utilities.social.view
+  (:require [quo.components.utilities.social.loader :as social-loader]
+            [react-native.core :as rn]
+            [schema.core :as schema]
+            [utils.number]))
+
+(def ?schema
+  [:=>
+   [:cat
+    [:map {:closed true}
+     [:accessibility-label {:optional true} [:or keyword? string?]]
+     [:size {:optional true :default 32} [:or keyword? pos-int?]]
```

Comment:
This part of the schema is not showing the proper default value. In the component the default is 20